gulpfile.js

```diff
@@ -90,7 +92,7 @@ gulp.task('test.compile', ['compile'], () => {
 });
 
 gulp.task('test.unit', ['compile'], cb => {
-  spawn('bash', ['./bin/run-test.sh'], {
+  spawn('bash', ['./bin/run-test.sh'].concat(taskArgs), {
```

Comment:
It may be good to have a more general mechanism for arg parsing if we end up with many gulp tasks that require arguments. Since there is currently only one task which has one optional argument, maybe it would be cleaner to have separate gulp tasks for `test.unit` and `coverage`?